lib/backpex/live_components/form_component.ex

```diff
@@ -179,7 +179,9 @@ defmodule Backpex.FormComponent do
   end
 
   def handle_event("save", _params, socket) do
-    handle_item_action(socket, nil, %{})
+    change = put_upload_change(%{}, socket, :insert)
+
+    handle_save(socket, socket.assigns.live_action, change)
   end
```

Comment:
Do you know why we handled an item action in the *catch-all* save event? I now handle a normal save. This fixes the bug addressed in the linked issue and everything works as expected. @pehbehbeh

Reply:
Also, I don't know if we really need to call the `put_upload_change` 🤔

Reply:
Sounds good then. 🤷 Would be nice to have tests to be sure. 😉